common/kubelet-wrapper

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -e
+# Wrapper for launching kubelet via docker.
+
+SYSTEMD_SERVICE_NAME="$1"; shift
+
+function require_ev_all() {
+	for rev in $@ ; do
+		if [[ -z "${!rev}" ]]; then
+			echo "${rev}" is not set
+			exit 1
+		fi
+	done
+}
+
+require_ev_all SYSTEMD_SERVICE_NAME KUBELET_IMAGE CLUSTER_DOMAIN_SUFFIX CLUSTER_DNS_SERVICE_IP
+
+mkdir -p /etc/kubernetes /var/lib/docker /var/lib/kubelet /run/kubelet
+
+readonly docker_cgroup_driver="$(docker info -f '{{.CgroupDriver}}')"
+if [[ "$docker_cgroup_driver" = "systemd" ]] ; then
+	for i in $(cat /proc/cgroups |awk '{print $1}'|grep -v subsys_name) ; do
+		# systemd does not create the entries for all cgroup controllers (e.g. hugetlb)
+		mkdir --parents /sys/fs/cgroup/$i/kubepods.slice
+	done
```

Comment:
When `kubelet-wrapper` is called as part of the kubelet systemd service, the cgroup driver is set correctly by the configuration file `/etc/kubernetes/kubelet.config` instead of a flag (22fb201c1e21034e5f5e81dd0f63d633be66370c):


https://github.com/kinvolk/lokomotive-kubernetes/blob/e0fe86eb2c0701c141b58c700199067915207602/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl#L190-L202

Or do you suggest to move the code from `configure-kubelet-cgroup-driver` into `kubelet-wrapper` so that the script becomes self-sufficient?

Reply:
Ok, then it's already fine with configure-kubelet-cgroup-driver.